python/social_bot/tasks.py

```diff
@@ -940,14 +946,13 @@ def run(self):
             target=self._object,
             random_range=self._object_random_range,
             center_pos=np.array([0, 0]),
-            min_distance=self._min_distance,
-            height=0)
+            min_distance=self._min_distance)
         goal_pos = self._random_move_object(
             target=self._goal,
             random_range=self._place_to_random_range,
             center_pos=obj_pos[:2],
             min_distance=self._min_distance,
-            height=0)
+            height=self._object_height)
```

Comment:
can you obtain the height using get_pose() instead of manually proving it?

Reply:
If we obtain height from get_pose() it would be zero. It's an "offset" related to model. i.e., half of height of the cube. e.g., 2.5cm for 5cm cube and 5cm for 10cm cube. 
The goal indicator (a small static red ball without collision) does not have offset, center of the ball is (0,0,0). That is to say if we set it to (0,0,0) half of the ball would be under ground. While the cube's center pos in the model has an 2.5cm offset, means If we set it to (0,0,0) it is properly on the ground. 

This manually option is for this offset.

But just found the model's offset cause another problem: if the cube is rotated, it is still on the ground but get_pose() will return a height of 5cm. The agent learns to filp it then grasp it without actually lifting the cube.  I think I'll remove the offset of the original cube model later as well as this option.